src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -2128,6 +2200,672 @@ namespace FanCoilUnits {
 	}
 
 	void
+		SimMultiStage4PipeFanCoil(
```

Comment:
@Nigusse If you are making changes, could you make sure this function name and the parentheses/brackets are indented uniformly?